### PR TITLE
Add --page parameter to projects and issues commands

### PR DESCRIPTION
**Pull request by erikthered**
Right now this just handles taking the flag value and setting the `page` query parameter to the API url. At this point I'm looking for feedback and feeling out the direction we want to go with this. 

The GitLab API does return a number of useful headers describing pagination state (next page, previous page, total records, etc.), but these are currently only available in `api.go`. I'm curious if you had anything in mind for how to get this info back to the commands themselves, since right now only the response body is getting sent back.

### CODE REVIEW

File: cmd/projects/projects.go [hunk 1]
```diff
@@ -14,8 +14,13 @@ import (
...
 
-func projectsCommand(client api.Client, cfg config.Config, quiet bool, format string, out io.Writer) error {
-	resp, status, err := client.Get("/users/${user}/projects")
+func projectsCommand(client api.Client, cfg config.Config, quiet bool, format string, page int, out io.Writer) error {
+	path := "/users/${user}/projects"
+	if page > 0 {
+		path += fmt.Sprintf("?page=%d", page)
+	}
+
+	resp, status, err := client.Get(path)
```
[the remark this answers is not in the document]
Comment (makkes) [reply cmd/projects/projects.go]: I think as a start it'd be ok to only have it for the `projects` command. I thought we could get some more general support for pagination to all commands but that's probably a bit too much for the time being.

**Comment by makkes** [comment]
@erikthered If you're fine with this I'd merge it right away

**Comment by erikthered** [comment]
I've got paging working for issues locally ise you want me to include that
too

**Comment by makkes** [comment]
> I've got paging working for issues locally ifyou want me to include that too.

let's include it. 👍

**Comment by erikthered** [comment]
Ok, this should be set to merge now

**Comment by makkes** [comment]
Please try not to force-push next time, it's a little hard to follow changes.
